CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - We added automatic browser extension install on Windows for Chrome and Edge. [#6076](https://github.com/JabRef/jabref/issues/6076)
 - We added a search bar for filtering keyboard shortcuts. [#11686](https://github.com/JabRef/jabref/issues/11686)
 - By double clicking on a local citation in the Citation Relations Tab you can now jump the linked entry. [#11955](https://github.com/JabRef/jabref/pull/11955)
+- When dragging and dropping several PDF files into the main table, the menu icon for background tasks becomes a progress indicator visualising the import's progress. [#12072](https://github.com/JabRef/jabref/pull/12072)
```

Comment:
Maybe explicitly say you added a progress indicator i.e. "We added a progress indicator that visualises an import's progress after dragging and dropping several PDF files...." etc

Reply:
I will refactor the comment to be more precise and follow the existing comments. Thank you!